shaders/src/main/glsl/samples/310es/stable_rects.frag

```diff
@@ -0,0 +1,65 @@
+#version 310 es
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+uniform vec2 resolution;
+
+const vec4 pal[16] = vec4[16](
+    vec4(0.0,0.0,0.0,0.0),
+    vec4(0.5,0.0,0.0,0.0),
+    vec4(0.0,0.5,0.0,0.0),    
+    vec4(0.5,0.5,0.0,0.0),
+    vec4(0.0,0.0,0.5,0.0),
+    vec4(0.5,0.0,0.5,0.0),    
+    vec4(0.0,0.5,0.5,0.0),    
+    vec4(0.5,0.5,0.5,0.0),
+    
+    vec4(0.0,0.0,0.0,0.0),
+    vec4(1.0,0.0,0.0,0.0),
+    vec4(0.0,1.0,0.0,0.0),
+    vec4(1.0,1.0,0.0,0.0),
+    vec4(0.0,0.0,1.0,0.0),
+    vec4(1.0,0.0,1.0,0.0),
+    vec4(0.0,1.0,1.0,0.0),
+    vec4(1.0,1.0,1.0,0.0));
+
+const vec4 picdata[] = vec4[](
+    vec4(4,4,20,4),

```

Comment:
Can you make the array size explicit, to keep GraphicsFuzz happy?

Reply:
done